java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -131,7 +137,7 @@ public void interruptedExecution(CheckerContext context) {
     public SEIssue(Tree tree, String message, Set<List<JavaFileScannerContext.Location>> flows) {
       this.tree = tree;
       this.message = message;
-      this.flows = flows;
+      this.flows = new HashSet<>(flows);
```

Comment:
AFAI understand : this line is the sole purpose of this commit. If we need to be defensive it could be great to see a test about it.